README.md

```diff
@@ -83,10 +84,198 @@ the build script. Then, run the test script in the root directory. It
+        p2 = re.compile('Repl\s*' + ''.join(['([0-9\.]+\s*)' for _ in range(replica_number - 1)]) + '$')
+        ready = False
+        answer=-1
+        for line in f.readlines():
+            if not ready and p1.findall(line):
+                ready = True
+            elif ready:
+                match = p2.match(line)
+                
+                if match:
+                    answer= [float(s) for s in match.groups()]
+                    break
+            
+        return answer
+```
+
+Specify PTE inputs and generate plumed file
+```python
+pte_result = ps.pte_replica(mpi_np=MPI_NP, max_tries=3,min_iters=1, mdp_kwargs={'nsteps':int(400* 5*10**2), }, replicas=replicas,hills_file_location=os.getcwd(),hot=400,hill_height=0.6,sigma=250,bias_factor=16, eff_threshold=0.20,cold=278,exchange_period=200)
+pte_plumed_script=pte_result['plumed']
+replica_temps=pte_result['temperatures']
+with open('plumed_pte.dat', 'w') as f:
+    f.write(pte_plumed_script)
+ps.add_file('plumed_pte.dat')
+temps = ','.join(str(e) for e in replica_temps)
+kwargs = [ {'ref_t': ti} for ti in replica_temps]
+time_ns=5.0 
+replex_eff = 0.2
+for kw in kwargs:
+    kw['nsteps']=  int(time_ns * 5 * 10 ** 5)
+max_iterations=20
+min_iterations=10
+```
+
+If you are running EDS or require additional output from plumed, you can add additional plumed script by
+```python
+plumed_input0=textwrap.dedent(
+    '''
+    plumed script
+    ''')
+plumed_input=pte_plumed_script+plumed_input0
+with open('plumed_eds_conver_pt_wte_metad.dat', 'w') as f:
+    f.write(plumed_input)
+ps.add_file('plumed_eds_conver_pt_wte_metad.dat')
+```
+
+Code for running simulation.
+```python
+for i in range(max_iterations):
+    ps.run(mdpfile='peptidesim_nvt.mdp', tag='nvt_conver_{}'.format(i),  
+           mdp_kwargs=kwargs, run_kwargs={'plumed':'plumed.dat','replex':remd_exhcange_period}, mpi_np=MPI_NP)
+    with open(ps.pickle_name, 'wb') as f:
+        pickle.dump(ps, file=f)
+
+    rep_eff_1 = get_replex_e(ps, replicas)
+    if rep_eff_1 ==-1:
+        ps.run(mdpfile='peptidesim_nvt.mdp', tag='nvt_conver_{}'.format(i),  mdp_kwargs=kwargs,mpi_np=MPI_NP, run_kwargs={'plumed':'plumed.dat','replex':remd_exhcange_period},mpi_np=MPI_NP)
+        with open(ps.pickle_name, 'wb') as f:
+            pickle.dump(ps, file=f)
+        
+    elif (min(rep_eff_1) >= replex_eff and i>=min_iterations):
+        print 'Reached replica exchange efficiency of {}. Continuing to production'.format(rep_eff_1)
+        break
+
+        with open(ps.pickle_name, 'wb') as f:
+            pickle.dump(ps, file=f)
+    else:
+        print 'Replica exchange efficiency of {}. Continuing simulation'.format(rep_eff_1)
+
+
+final_time_eds=int(0.040*5*10**5)
+ps.run(mdpfile='peptidesim_nvt.mdp', tag='nvt_prod',  mdp_kwargs={'nsteps': final_time_eds, 'ref_t': 278},mpi_np=MPI_NP)
+with open(ps.pickle_name, 'wb') as f:
+    pickle.dump(ps, file=f)
+#finally:
+print('done')
```

Comment:
Do we really need to print 'done' in the documentation?

Reply:
removed in the new commit